HW1/P3/P3.py

```diff
@@ -0,0 +1,18 @@
+import findspark
+findspark.init()
+import pyspark
+sc = pyspark.SparkContext(appName='anagrams')
+
+with open('EOWL_words.txt', 'r') as rfile:
+	words = rfile.read().split('\n')
+
+lex_order = lambda s: ''.join(sorted(list(s)))
+
+words_rdd = sc.parallelize(words, 10)
+lex_to_word = words_rdd.map(lambda w: (lex_order(w), w))
+lex_to_wordlist = lex_to_word.groupByKey()#.mapValues(list)
```

Comment:
Try to avoid using groupByKey() whenever you can (for efficiency). A reduceByKey() may have been the better call here.